front/app/utils/streams/streams.test.ts

```diff
@@ -0,0 +1,528 @@
+import { Streams } from '.';
+// @ts-ignore
```

Comment:
Is it safe to ignore ts here?

Reply:
It only ignores the next line. TS gets confused by the mock for some reason... if you have any idea why and how to fix it I'd be happy to change it but I couldn't figure it out

Reply:
Yeah, I think it's acceptable in this case, if it would take too long to solve. 👍